src/gt4sd/training_pipelines/tests/test_training_moses_vae.py

```diff
@@ -23,19 +23,21 @@
 #
 """Moses VAE trainer unit tests."""
 
+import atexit
 import os
 import shutil
 import tempfile
+from contextlib import ExitStack
 from typing import Any, Dict, cast
 
-import pkg_resources
+import importlib_resources
 
 from gt4sd.training_pipelines import TRAINING_PIPELINE_MAPPING, MosesVAETrainingPipeline
 
-TEST_DATA_DIRECTORY = pkg_resources.resource_filename(
-    "gt4sd",
-    "training_pipelines/tests/",
-)
+file_manager = ExitStack()
```

Comment:
Let's extract a method as the only thing that is changing is the file path

Reply:
Done